ThemingCodelab/app/src/main/java/com/codelab/theming/ui/start/Home.kt

```diff
@@ -115,6 +126,7 @@ fun FeaturedPost(
         ) {
             Image(
                 bitmap = imageResource(post.imageId),
+                contentDescription = stringResource(R.string.label_post_image),
```

Comment:
This image is currently used only for decorative purposes and does not represent a meaningful action that a user can take. Therefore you want to set its contentDescription to `null`.